src/main/java/io/confluent/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -218,7 +216,11 @@ public BufferedRecords(
    */
   private Optional<Long> executeUpdates() throws SQLException {
     Optional<Long> count = Optional.empty();
+
     for (int updateCount : updatePreparedStatement.executeBatch()) {
+      if (updateCount == Statement.EXECUTE_FAILED) {
+        throw new BatchUpdateException("Execution failed for part of the batch update", new int[]{Statement.EXECUTE_FAILED});
```

Comment:
can we just include `updatePreparedStatement.executeBatch()` for the updateCount array?